src/client/containers/QuizPage/QuizPage.component.js

```diff
@@ -1,3 +1,45 @@
 import React from 'react';
+import './QuizPage.styles.css';
+import PageHeader from '../../components/PageHeaderComponent/PageHeader';
+import QuestionComponent from '../../components/Question/QuestionComponent';
+import QuizAnswers from '../../components/QuizAnswers/QuizAnswers';
+import SideMenu from '../../components/sideMenubar/sideMenu';
+import Buttons from '../../components/Buttons/Buttons';
+import ProgressBar from '../../components/ProgressBar.components/ProgressBar';
+import image1 from '../../assets/images/questionBackgrounds/question1background.png';
 
-export const QuizPage = () => <section>Quiz Page</section>;
+export const QuizPage = () => {
+  return (
+    <div>
+      <div className="main">
+        <div className="header_page">
+          <PageHeader />
+        </div>
+        <div className="middle_page">
+          <div className="sidemenu_page">
+            <SideMenu highLigtItem={3} />
```

Comment:
I think there is a typo here `highLightItem={3}` missing an `h`

Reply:
Thanks for review.. done